app/assets/javascripts/organization.coffee

```diff
@@ -0,0 +1,19 @@
+window.addAddressRow = (count) ->
+  newRow = $ """
+  <div class="form-group">
+    <label for="organization_addresses_attributes_""" + count + """_address">Mailing Address</label>
+
+    <div>
+      <input class="form-control" data-guard="different" type="text" value="" name="organization[addresses_attributes][""" + count + """][address]" id="organization_addresses_attributes_""" + count + """_address">
```

Comment:
I appreciate the reason for this but this code is written to match the generated helper code. If we want to go a different direction let's do it after 0.1 release I think.